test/tests/functional/pbs_qstat_validate_attr.py

```diff
@@ -73,45 +81,34 @@ def validate_el(self, attr_dict, user):
 
         for each in attr_dict:
             if each in qstat_o:
-                if "count" in each:
+                if "state_count" in each:
                     out1 = []
                     out = attr_dict[each].split()
                     for k in out:
                         out1.append(k.split(':')[0])
-                    self.assertListEqual(qstat_o[each], out1)
+                    self.assertListEqual(qstat_o[each], out1,
+                                         "Value of server attribute \"%s\""
+                                         "does not match expected"
+                                         "value" % each)
                 else:
-                    self.assertEqual(qstat_o[each], attr_dict[each])
-
-    @staticmethod
-    def filter_el(e):
-        for each in e:
-            if "=" in each:
-                return each
+                    self.assertEqual(qstat_o[each], attr_dict[each],
+                                     "Value of server attribute \"%s\""
+                                     "does not match expected value" % each)
+        self.logger.info(
```

Comment:
Please also add a log message inside the loop printing what values are successfully matched. for example, self.logger.info ("successfully verified that qstat_o[each] is same as attr_dict[each]")